src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
+Such a partner deploys and supports an infrastructure based on Juju. We call
+this a *managed solution*.
 
-There are three support levels that may be set on a model:
+Any support requests beyond the build, workload, and operation of a particular
+solution can be escalated to Canonical by means of their Ubuntu Advantage
+support framework. Prime examples of such requests are for issues relating to
+Juju itself, Ubuntu-specific software, or the operating system kernel. The
+escalated support case will be treated by Canonical at the same support level
+currently ascribed to the partner. Hence, support level names used by both a
+Juju Expert and by Canonical are the same:
 
-- Essential
-- Standard
-- Advanced
+ - Essential
+ - Standard
+ - Advanced
 
-Our Juju expert partners support the managed solution itself, including its
-workloads and operation. Support for Juju and Ubuntu is provided by
-Canonical. See [Ubuntu Advantage for Servers](https://www.ubuntu.com/support/plans-and-pricing#server)
-for more information on Canonical support levels.
+See [Ubuntu Advantage for Servers][ubuntu-advantage-servers] for more
+information on Canonical support levels.
 
-Juju experts will provide details on the solution-specific support provided
-under each SLA, with budget estimates.
+The Juju Expert will provide details such as support levels and costs on a
+per-solution basis but ultimately an SLA will be conferred upon a specific Juju
+model (see [Models][models]).
 
-## Setting an SLA on a model
+Consult the external [Juju Experts][juju-experts] page to find a managed
+solution partner for your project.
 
-An SLA is set on a model with a monthly budget limit. Support costs will not be
-billed for more than this amount. If metered costs exceed this amount, the
-budget limit will need to be increased in order to continue to access support.
+## Requirements
 
-Set an Essential SLA on the current model _marketdata_, with a monthly budget
-limit of $100 USD:
+The following requirements need to be met before setting an SLA for the first
+time:
+
+ 1. The Juju operator will need an [Ubuntu SSO][ubuntu-sso] account.
+ 1. The Juju operator will need to have logged in to [Launchpad][launchpad]
+    (via Ubuntu SSO) at least once.
+
+## Setting an SLA
+
+Once you have contracted out a Juju Expert you are ready to set an SLA. This is
+done within Juju itself via the `sla` command, which will trigger a dialog to
+authenticate to your Ubuntu SSO account.
+
+An SLA is set on a per-model basis, and includes two key parameters in order
+for support to become active:
+
+ 1. Support level
+ 1. Budget limit
+
+Support costs will not be billed for more than the budget limit. If metered
+costs exceed this amount, the limit will need to be increased in order for
+support to remain active.
+
+For example, to set an Essential SLA on model 'default', with a monthly budget
+limit of USD 100:
 
 ```bash
-juju sla essential --budget 100
+juju sla -m default essential --budget 100
```

Comment:
can we use a different model name vs default for the example. It's almost certainly not "default" in any real case and might confuse folks.

Reply:
This was deliberate and is often used throughout the documentation. The base documentation should not be using exotic names when introducing new topics. Tutorials and blogs and documented use cases are better for that.

Reply:
k